ext/scheduler/airflow2/airflow.go

```diff
@@ -213,18 +220,16 @@ func (s *scheduler) GetJobStatus(ctx context.Context, projSpec models.ProjectSpe
 	}
 
 	fetchURL := fmt.Sprintf(fmt.Sprintf("%s/%s", schdHost, dagStatusUrl), jobName)
```

Comment:
Can we merge calls to multiple `Sprintf`

Reply:
removed fmt.Sprintf() at latest commit